frame/support/src/storage/mod.rs

```diff
@@ -17,6 +17,7 @@
 
 //! Stuff to do with the runtime's storage.
 
+pub use self::types::StorageEntryMetadata;
```

Comment:
I would prefer if macro use `frame_support::metadata::StorageEntryMetadata` than do a reexport here. 

Is there a reason for this reexport here ?
Also the reexport of metadata from frame-support doesn't need to be hidden IMHO.

Anyway it is not a blocker just a nitpick, that can be solved in a follow-up